js/main.js

```diff
@@ -34,24 +40,36 @@
       //タスクボタンに状態を添付する処理
       createStatus(task, tr)
       //削除ボタンに削除と添付する処理
-      createDelete(tr)
+      createDelete(tr,  index)
```

Comment:
細かいですが、スペースが空きすぎていますので修正しましょう！
```suggestion
      createDelete(tr, index)
```